src/uu/chown/src/chown.rs

```diff
@@ -131,10 +139,7 @@ pub fn uumain(args: impl uucore::Args) -> i32 {
             Ok((None, Some(gid))) => IfFrom::Group(gid),
             Ok((Some(uid), Some(gid))) => IfFrom::UserGroup(uid, gid),
             Ok((None, None)) => IfFrom::All,
-            Err(e) => {
-                show_error!("{}", e);
-                return 1;
-            }
+            Err(e) => return Err(USimpleError::new(1, e)),
```

Comment:
Ideally, `parse_spec` would return `UResult`, so you don't need to create a custom error every time you call it. For example, this block would become:
```rust
match parse_spec(spec)? {
    (Some(uid), None) => IfFrom::User(uid),
    (None, Some(gid)) => IfFrom::Group(gid),
    (Some(uid), Some(gid)) => IfFrom::UserGroup(uid, gid),
    (None, None) => IfFrom::All,
}
```